workers/unity/Packages/com.improbable.gdk.deploymentlauncher/.DeploymentLauncher/Options.cs

```diff
@@ -0,0 +1,65 @@
+using System.Collections.Generic;
+using CommandLine;
+
+namespace Improbable.Gdk.DeploymentLauncher
+{
+    public class Options
+    {
+        [Verb("create", HelpText = "Create a deployment.")]
+        public class Create
+        {
+            [Option("project_name", Required = true, HelpText = "The SpatialOS project name")]
+            public string ProjectName { get; set; }
+
+            [Option("assembly_name", Required = true, HelpText = "The assembly to use in the deployment.")]
+            public string AssemblyName { get; set; }
+
+            [Option("deployment_name", Required = true, HelpText = "The name of the deployment to create.")]
+            public string DeploymentName { get; set; }
+
+            [Option("launch_json_path", Required = true, HelpText = "The path to the launch json file.")]
+            public string LaunchJsonPath { get; set; }
+
+            [Option("snapshot_path", Required = false, HelpText = "The path to the snapshot.")]
+            public string SnapshotPath { get; set; }
+
+            [Option("region", Required = true, HelpText = "The region to launch the deployment in.")]
+            public string Region { get; set; }
+
+            [Option("tags", Required = false, HelpText = "Tags to add to this deployment. Comma separated",
+                Separator = ',')]
+            public IEnumerable<string> Tags { get; set; }
+        }
+
+        [Verb("create-sim", HelpText = "Create a simulated player deployment")]
```

Comment:
Is `create-sim` too brief?

Reply:
Hmm possibly? `create-simulated-player` is simultaneously ambiguous in what it does and too verbose though 😕